### ISSUE TITLE
Add a `dayFormat` prop to customize "newDay"

### ISSUE THREAD

**Issue by zedix** [mariomka/vue-datetime#21]
**Feature Request:**
Allow to customize the "newDay" format.

The reason is that the default format "ddd, MMM D" does not make much sense for french people in example (almost not readable). It would be useful to customize it.

Here is possible new prop `dayFormat`:

```js
props: {  
    dayFormat: {
        type: String,
        default: 'ddd, MMM D'
    },
}
...
newDay () {
    return this.newDate.format(this.dayFormat)
}
```

Thanks you,

**Comment by mariomka** [comment]
Yes, this new prop will be useful. I can implement when I have time or if you want to do a pull request feel free.
